cppForSwig/cryptopp/Makefile

```diff
@@ -92,9 +93,11 @@ endif
 endif
 
 ifeq ($(UNAME),Darwin)
-AR = libtool
-ARFLAGS = -static -o
-CXX = c++
+# JB: Fix this so c++ is used if compiling natively
```

Comment:
I'm pretty sure c++ = clang++, so this comment shouldn't be necessary.

Also, why is the archive stuff commented out? Were errors popping up?